crates/mir/src/ir/types.rs

```diff
@@ -67,7 +66,7 @@ pub struct TupleDef {
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct StructDef {
     pub name: SmolStr,
-    pub fields: Vec<(SmolStr, TypeId)>,
+    pub fields: Vec<(SmolStr, Vec<SmolStr>, TypeId)>,
```

Comment:
I don't think we should add attributes to MIR types.

Reply:
Ah, ok. :+1:  Probably no big deal because we can lookup the analyzer type.

Reply:
Removed them.